src/test/java/teammates/test/cases/browsertests/InstructorStudentRecordsPageUiTest.java

```diff
@@ -63,18 +63,6 @@ private void testContent() throws Exception {
         viewPage = getStudentRecordsPage();
         viewPage.verifyHtmlMainContent("/instructorStudentRecordsWithHelperView.html");
 
-        ______TS("content: normal student records with other instructor's comments, private feedback session");
```

Comment:
I believe this is directly related to private sessions, and hence need to be deleted. right?

Reply:
What about the `normal student records with other instructor's comments` part? I'm guessing that the comments are not part of the private session?

Reply:
You were right. I misinterpreted this test.